Dockerfile

```diff
@@ -1,6 +1,10 @@
 FROM ubuntu:18.04
 ARG DEBIAN_FRONTEND=noninteractive
 
+# Avoid dropping man pages
+RUN sed --expression '/path-exclude=\/usr\/share\/doc\/\*/ s/^#*/#/' --in-place /etc/dpkg/dpkg.cfg.d/excludes && \
+    sed --expression '/path-exclude=\/usr\/share\/man\/\*/ s/^#*/#/' --in-place /etc/dpkg/dpkg.cfg.d/excludes
```

Comment:
For robustness, better to ensure you match the whole line, including any leading whitespace. Probably fine to just match on

```
\s*path-exclude=.*
```
no?

Reply:
Matching whole line per 74276a3. Honestly, I'd rather be more specific here to avoid accidentally excluding something that's not meant to be excluded?